pom.xml

```diff
@@ -23,6 +23,89 @@
         <module>cp-ksqldb-examples</module>
     </modules>
 
+    <dependencyManagement>
```

Comment:
Will `<dependencyManagement>` going to add these netty versions to the classpath? Don't other modules need to call the netty dependency too?

Reply:
No, it's not going to do this. `dependencyManagment` just sets the version of transitive dependencies, but if there is no dependency to the given maven artifact, it will have no effect.